README.md

```diff
@@ -7,7 +7,8 @@
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
 ```text
-Your answer...
+SQL DB is Relational database with tables and predefined schemas whereas NoSQL is non Relational database which doesnt have table models and has key value pairs. PSQL is SQLDB and MongoDB is NoSQL.
+I will use SQL when i know what data type is expected and will use NoSQL when im not sure of datatypes, i can get this DB started quickly.
 
 ```
```

Comment:
What do you mean by "predefined schemas" in SQL? You need to set those up yourself.